README.md

```diff
@@ -1,221 +1,139 @@
-# Uploadcare TinyMCE Plugin
+# File Uploader by Uploadcare
+
+<a href="https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-tinymce">
+  <img align="right" width="64" height="64"
+    src="https://ucarecdn.com/2f4864b7-ed0e-4411-965b-8148623aa680/uploadcare-logo-mark.svg"
+    alt="">
+</a>
+
+This is a plugin for the [TinyMCE][tinymce] WYSIWYG HTML editor
+providing it to work with [Uploadcare Widget][uc-feature-widget].
```

Comment:
This is a plugin for TinyMCE, a WYSIWYG HTML editor, providing it for working with Uploadcare Widget.